### ISSUE TITLE
Types missing, need a new version pushed?

### ISSUE THREAD

**Issue by thequailman** [MithrilJS/mithril-query#114]
Hello, I am trying to use mithril-query with types but it looks like the latest version on NPM (4.0.1) doesn't include the `types` property in package.json.  I'm not sure what commit built it, but could you push a new version to NPM with the current code in master?

For now, I'm using `"mithril-query": "git+https://github.com/mithriljs/mithril-query.git#facf7ce"` in my package.json and it works OK.

**Comment by mindreframer** [comment]
@StephanHoyer Yes, the published version on NPM does not have the "types" property in package.json. Just added it manually and it fixes the typing problem. And since it's already present in this repo, publishing a 4.0.2 with proper package.json would fix that issue. 

Please consider doing it.

**Comment by JodliDev** [comment]
@dead-claudia, @StephanHoyer

It has been 4 years since the npm repository has been updated and is still not up to date with github. Is this project still being maintained? :)

**Comment by dead-claudia** [comment]
I'll look into publishing a new version first chance I get (which won't be for a bit).

I need to integrate the same auto-publish system that we have for the core Mithril repo. It'll take some effort, and has been somewhat low on my TODO list. I've pulled out all the common stuff already, but for some reason it's seeing a wrong repo somewhere down the line, and I need to investigate before I can make that transition for everything.
